packages/ui-library/src/theme/variants/select-variants.ts

```diff
@@ -30,34 +34,30 @@ export const buildSelectVariant = (mode: 'light' | 'dark'): Components => ({
         '&.MuiSelect-MuiMenu': {
           top: SPACING(2),
           boxShadow: colors[mode].dropShadow.dropShadow200,
-          '& .MuiMenu-paper': {
-            backgroundColor: colors[mode].background.secondary,
-            border: `1px solid ${colors[mode].border.border2}`,
-            '& .MuiMenu-list': {
-              padding: `${SPACING(3)} !important`,
-              gap: SPACING(0.5),
-              '& .MuiDivider-root': {
-                margin: `${SPACING(4)} ${SPACING(0)}`,
-              },
-              '& .MuiMenuItem-root': {
-                padding: SPACING(3),
-                borderRadius: SPACING(2),
-                border: `1px solid ${colors[mode].border.border1}`,
-                backgroundColor: colors[mode].background.secondary,
-                '&:hover': {
-                  backgroundColor: colors[mode].background.emphasis,
-                },
-              },
-              '& .Mui-selected': {
-                padding: SPACING(3),
+          '& .MuiMenu-paper .MuiMenu-list': {
+            padding: `${SPACING(3)} !important`,
+            gap: SPACING(0.5),
+            '& .MuiDivider-root': {
+              margin: `${SPACING(4)} ${SPACING(0)}`,
+            },
+            '& .MuiMenuItem-root': {
+              padding: SPACING(3),
+              borderRadius: SPACING(2),
+              border: `1px solid ${colors[mode].border.border1}`,
+              backgroundColor: colors[mode].background.secondary,
+              '&:hover': {
                 backgroundColor: colors[mode].background.emphasis,
               },
             },
           },
+          '& .Mui-selected': {
+            padding: SPACING(3),
+            backgroundColor: colors[mode].background.emphasis,
+          },
         },
       },
       paper: {
-        backgroundColor: colors[mode].background.secondary,
+        backgroundColor: colors[mode].background.tertiary,
```

Comment:
In this file we had some duplicated styles, also this bg was not matching design for our selects
This is in prod with bg-color: secondary:
<img width="603" alt="Screenshot 2024-12-02 at 11 02 18 AM" src="https://github.com/user-attachments/assets/61ed3e19-acd5-40c4-b9b3-665c2d19c6ff">